addons/account/wizard/account_move_send.py

```diff
@@ -147,6 +164,7 @@ def _get_wizard_values(self):
             'sp_user_id': self.env.user.id,
             'download': self.checkbox_download,
             'send_mail': self.checkbox_send_mail,
+            'pdf_template_id': self.pdf_template_id,
```

Comment:
this must be serializables data that could be stored on the account.move json field so
```suggestion
            'pdf_report_name': self.pdf_template_id.report_name,
```
see:
https://github.com/odoo/odoo/blob/17a7c7ee9f65190e046779b645375d0a18bcba84/addons/account/wizard/account_move_send.py#L761